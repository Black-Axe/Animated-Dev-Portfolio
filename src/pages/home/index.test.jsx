import React from 'react'
import { expect, it } from 'vitest'
import HomePage from '.'

it('renders snapshot of home page', () => {
  expect(<HomePage />).toMatchSnapshot()
})
