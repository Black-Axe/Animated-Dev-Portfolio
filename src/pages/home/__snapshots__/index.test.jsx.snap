// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders snapshot of home page 1`] = `<HomePage />`;
